.changeset/chilled-pandas-approve.md

```diff
@@ -0,0 +1,5 @@
+---
+"@turnkey/sdk-react": minor
+---
+
+Update default country codes to supported deliveries
```

Comment:
Should we include the actual list? I'm not sure our current changeset/versioning system allows readers to easily reference the commit and find the changes to `packages/sdk-react/src/components/auth/PhoneInput.tsx`